### ISSUE TITLE
Update camera library to latest version

### ISSUE THREAD

**Issue by mariobehling** [fossasia/phimpme-android#2859]
If we update the camera library to a newer version it could help to close a number of bugs.

**Comment by codedsun** [comment]
@yashk2000 what has to be done in this. please describe

**Comment by yashk2000** [comment]
Currently, you'll see that we have a module called `open camera`. This version of open camera being used is very old. It has been updated a lot as you can see [here](https://sourceforge.net/projects/opencamera/). 
We need to update our module with the new module. This blog might help: https://blog.fossasia.org/phimpme-merging-several-android-projects-into-one-project/

**Comment by codedsun** [comment]
@yashk2000 that means we will take the files from the latest release and merge that in our project, as we did that earlier?

**Comment by yashk2000** [comment]
Yes. We'll have to repeat that entire process with the updated files.

**Comment by codedsun** [comment]
@yashk2000 Do we also need to add extra features of video editing?

**Comment by yashk2000** [comment]
@iamareebjamal should we go ahead with adding features of video editing too? 

If not video editing, I think we should still let users take videos using phimpme camera and view the videos in phimpme gallery instead of making a user switch between multiple apps to record and view videos. What are your views on this @iamareebjamal ?

**Comment by iamareebjamal** [comment]
First, upgrade with only existing functionality and make it big free

**Comment by codedsun** [comment]
@iamareebjamal Okay, I will do it.

**Comment by codedsun** [comment]
@iamareebjamal I am copying the files from the latest apk, and making changes into the current prooject. It requires a lot of changes and is taking time, how will I send the PR? step by step or once  all done?

**Comment by iamareebjamal** [comment]
I don't understand

**Comment by codedsun** [comment]
@iamareebjamal What I meant to say that
1. I am solving this issue that means I am merging opencamera to phimpme project which requires me to copy the source code of opencamera project to phimpme

2. The above task requires a lot changes in the project and the module named `opencamera`, so the concern is shall I send a direct PR with all the commits at once or I send a PR now and do the commits regularly?

**Comment by iamareebjamal** [comment]
Why are you merging it and not using it as a library?

**Comment by codedsun** [comment]
@iamareebjamal The blog states this only
https://blog.fossasia.org/phimpme-merging-several-android-projects-into-one-project/


Also i see no jar file to import from their project, please guide me further

**Comment by codedsun** [comment]
@iamareebjamal Any update?

**Comment by iamareebjamal** [comment]
Do it in 1 PR
